document_definition/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+document_definition
+===================
+
+Installs a new  document_definition model for dictionary-style word archive and
+definition. Searchable in the Knowledge menu.
+
+Known issues / Roadmap
+======================
+
+* A search function per term and per definition, obviously.
+* Dictionary Style Alphabetical grouping view
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/knowledge/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Giovanni Francesco Capalbo <giovanni@therp.nl>  
```

Comment:
Isnt this gcapalbo?

Reply:
github handle gfcapalbo, shouldn't the proper name be here?